src/store/view-configs/volume-coloring.ts

```diff
@@ -91,26 +111,41 @@ export const setupVolumeColorConfig = () => {
   const updateVolumeOpacityFunction = createUpdateFunc('opacityFunction');
   const updateVolumeCVRParameters = createUpdateFunc('cvr');
 
+  const setVolumeColorPreset = (
+    viewID: string,
+    imageID: string,
+    preset: string
+  ) => {
+    const imageStore = useImageStore();
+    const image = imageStore.dataIndex[imageID];
+    if (!image) return;
```

Comment:
`console.warn()` here?  Or comment on case when we expect no image?

Reply:
There isn't much of a chance that this would occur. Realistically, throwing would be more appropriate here, but the one valid case is if the dataset somehow gets deleted prior to setVolumeColoPreset (e.g. in a promise).

Reply:
> Good defaults are nice. Did we want to filter the big list of presets in this PR or save for later?

Good call on the filtering. Do we want to filter @aylward? I think it would be useful to have all presets available regardless of the modality.